shared/js/numerals_helper.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+(function(exports) {
+  var NumeralsHelper = {
+    _cnvMutations: function() {
+      this._applyEAWA();
+    },
+
+    _applyEAWA: function() {
+      var lang = document.documentElement.lang;
+      if (lang === 'ar') {
+        this.toEA();
+      } else {
+        this.toWA();
+      }
+    },
+
+    _cnvEAWA: function(b) {
+      var allHTMLElements = [].slice
+        .call(document.querySelectorAll('[data-intl-numerals]'));
+      allHTMLElements.forEach(function(el) {
```

Comment:
Maybe call `forEach` right on the qSA?

    Array.prototype.forEach.call(document.querySelectorAll('[data-intl-numerals]'));